### PR TITLE
gimp: Update to 2.10.38 + rebuilds

### PR DESCRIPTION
**Pull request by davidjharder**
**Summary**

Release notes [here](https://www.gimp.org/news/2024/05/05/gimp-2-10-38-released/)

**Test Plan**

- Edit an image
- Try out some resythesizer plugin features

**Checklist**

- [x] Package was built and tested against unstable

Resolves https://github.com/getsolus/packages/issues/2350 thanks malfisya

### CODE REVIEW

File: packages/g/gimp/monitoring.yml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+releases:
+  id: 1161
+  # 2.99.* is the development tag (until version 3 exists)
+  igonore: "2.99.*"
```
Comment (ReillyBrogan) [inline packages/g/gimp/monitoring.yml:4]: Igonore

Comment (davidjharder) [reply packages/g/gimp/monitoring.yml]: fiixed

**Comment by malfisya** [comment]
Resolves https://github.com/getsolus/packages/issues/2350

**Review by ZachBacon — APPROVED** [review]
LGTM

**Pull request merged**